src/extraSettingPage/options.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>My Extension Options</title>
+</head>
+<body>
+    <label>
+        Click here to show more debug info:
```

Comment:
Just curious - can we use a check box option in the extension setting instead of a standalone page?

![Screenshot 2023-08-08 at 4 05 28 PM](https://github.com/GoogleChrome/audion/assets/676891/ac1072fa-071a-4372-a9fd-26e3de148ded)